### ISSUE TITLE
Reply method returns None

### ISSUE THREAD

**Issue by ibalazevic** [aichaos/rivescript-python#51]
For the template

```
+ [*] [what] [is|are|were|was] [_] (latest|new|hot off the press) [news|info|information] [_] [please|thanks|thank you|thank you very much]
- latest
```

and the input message "what is new" the reply method returns a None. 

Traceback:

```
File "basic_bot.py", line 42, in on_message
    reply = self.rs.reply(user, message)
  File "/Library/Python/2.7/site-packages/rivescript/rivescript.py", line 1786, in reply
    reply = self._getreply(user, msg, ignore_object_errors=errors_as_replies)
  File "/Library/Python/2.7/site-packages/rivescript/rivescript.py", line 2142, in _getreply
    reply = self._process_tags(user, msg, reply, stars, thatstars, step, ignore_object_errors)
  File "/Library/Python/2.7/site-packages/rivescript/rivescript.py", line 2391, in _process_tags
    reply = reply.replace('<star>', stars[1])
TypeError: coercing to Unicode: need string or buffer, NoneType found
```

**Comment by kirsle** [comment]
It seems you're on an old version of `rivescript-python`: the filename and line referenced in the exception is no longer in that place, but is now in `rivescript/brain.py` near line 579.

Bug #54 has the same problem you do, and I'm going to fix it and deploy a new version in a few minutes.

**Comment by kirsle** [comment]
This problem should be resolved if you update to v1.14.3.
